#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

uint32 size_prev
uint32 data_size
uint8 device_id
uint16 data_type
time timestamp